### PR TITLE
Export only SPDX identifiers, not SPDX expressions

### PR DESCRIPTION
**Pull request by kemitchell**
The SPDX JSON file at https://spdx.org/licenses/licenses.json currently includes both SPDX identifiers like "GPL-3.0" for "the GNU Public License version 3.0" and some expressions with the plus operator, like "GPL-3.0+" for "the GNU Public License version 3.0 or later". The latter is technically an SPDX expression with the identifier "GPL-3.0" and the "+" operator, and not an SPDX identifier.

This patch filters the elements of the licenses array in SPDX.org's JSON file to omit listings that end with "+", to omit those elements that are actually SPDX expressions with the "+" SPDX expression operator.

I've also rebuilt the files using the updated build script.

cc @goneall

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -13,7 +13,12 @@ const pkg = require('./package.json');
 got('https://spdx.org/licenses/licenses.json', {json: true}, (gotError, json) => {
   assert.ifError(gotError);
 
-  let array = json.licenses.map(license => license.licenseId);
+  let array = json.licenses
+    .filter(license => {
```
Comment (shinnn) [inline build.js]:
``` js
  .filter(license => !license.licenseId.endsWith('+'))
```

**Comment by shinnn** [comment]
:+1: Thanks, @kemitchell. I'll publish a new version soon.

**Comment by shinnn** [comment]
Released as v1.2.0. https://github.com/shinnn/spdx-license-ids/commit/f74a7a16ca05540e0e97f1bbb61da07829b5d9ab

**Comment by kemitchell** [comment]
Thanks, @shinnn!

**Pull request merged**